www/gatsby-node.js

```diff
@@ -815,3 +815,42 @@ exports.onPostBuild = () => {
     `./public/gatsbygram.mp4`
   )
 }
+
+// XXX this should probably be a plugin or something.
+exports.sourceNodes = ({ actions: { createTypes } }) => {
+  /*
+   * NOTE: This _only_ defines the schema we currently query for. If anything in
+   * the query at `src/pages/contributing/events.js` changes, we need to make
+   * sure these types are updated as well.
+   *
+   * But why?! Why would I do something this fragile?
+   *
+   * Gather round, children, and I’ll tell you the tale of @jlengstorf being too
+   * lazy to make upstream fixes...
+   */
+  const typeDefs = `
+    type AirtableConnection {
+      nodes: [Airtable!]!
+    }
+
+    type Airtable implements Node {
+      id: ID!
+      data: AirtableData
+    }
+
+    type AirtableData {
```

Comment:
```suggestion
    type AirtableEvent {
```

Reply:
These tables are auto-generated, so I don't think we get to rename them, unfortunately.